pkg/util/paramtable/base_table.go

```diff
@@ -43,6 +43,7 @@ const (
 	DefaultMinioCloudProvider   = "aws"
 	DefaultMinioIAMEndpoint     = ""
 	DefaultEtcdEndpoints        = "localhost:2379"
+	DefaultTiKVEndpoints        = "127.0.0.1:2379"
```

Comment:
Should we have DefaultKVEndpoints = "127.0.0.1:2379" instead of seperate but equal

Reply:
It's actually not needed yet. removing it.